src/custom.module.css

```diff
@@ -79,9 +79,9 @@
 }
 
 /* https://www.w3schools.com/cssref/css_selectors.asp */
-.actionButton:global(.pf-c-button.pf-m-control) {
-    padding-top: 0px;
-    padding-bottom: 0px;
+.actionButton:global(.pf-c-button) {
```

Comment:
Do we want to target different component?
Selector changes.

Reply:
Yes, the thing is that the "waive" and "rerun" buttons have the `pf-m-control` class, but the docs link has `pf-m-secondary`. I think it's OK to just target all buttons with the class `actionButton`.